### DISCUSSION TITLE
Driver not initiating in UC Mode in Google Colab

### DISCUSSION THREAD

**Discussion by abhishek-seo** [seleniumbase/SeleniumBase#2931] [Q&A] [answered]
Hi. I am trying to initiate the driver in UC mode in Google Colab but it is not working. It works in non-UC Mode. 

Here is my code - 

```
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys
import time
from seleniumbase import Driver
import random
import pandas as pd
from IPython.core.display import display, HTML
import pygsheets

driver = Driver(uc=True, headless=True)
```

And here is the error - 

SessionNotCreatedException: Message: session not created: cannot connect to chrome at 127.0.0.1:9222
from chrome not reachable

Though this code was working when I tried first, it stopped working suddenly after a day. 

What could be the reason?

**Comment by parth76** [comment]
I faced the same problem, Just install Google Chrome.

**Reply by abhishek-seo** [reply]
Yes I have installed chrome browser. It works in non-UC mode but does not work when I mark uc=True.
![image](https://github.com/user-attachments/assets/be807cd6-7912-4b0c-a8f7-d0addeea2fc5)

**Comment by mdmintz** [comment, marked as answer]
As mentioned in https://github.com/seleniumbase/SeleniumBase/discussions/2802#discussioncomment-9555175, you have to use regular chrome, not chromium. The binary would be called `google-chrome` or `google-chrome-stable` on Linux.

For UC Mode, you can no longer use Headless Mode if you want to remain undetected. With the virtual display that's included with the `SB(uc=True)` format, you can run non-headless Chrome on a machine with no display.

**Reply by abhishek-seo** [reply]
Thanks! That helped :)
